src/application/port/in/wishlist/WishlistUseCase.ts

```diff
@@ -0,0 +1,24 @@
+import { IsNumber, IsString } from 'class-validator';
+import { SwaggerDto } from 'src/common/decorators/swagger-dto.decorator';
+import { WishlistGroupedByReceiver } from 'src/domain/wishlist';
+
+@SwaggerDto()
+export class AddToWishlistRequest {
+  @IsNumber()
+  productId: number;
+
+  @IsString()
+  receiverName: string;
+}
+
+export interface AddWishlistCommand {
+  userId: number;
+  productId: number;
+  receiverName: string;
+}
+
+export interface WishlistUseCase {
+  addToWishlist(command: AddWishlistCommand): Promise<void>;
+  removeFromWishlist(wishlistId: number, userId: number): Promise<void>;
+  getGroupedByReceiver(userId: number): Promise<WishlistGroupedByReceiver[]>;
```

Comment:
인터페이스 작성 순서는 작성  -> 생성 -> 수정 -> 삭제 순으로 작성 부탁드릴게요
```suggestion
  getGroupedByReceiver(userId: number): Promise<WishlistGroupedByReceiver[]>;
  addToWishlist(command: AddWishlistCommand): Promise<void>;
  removeFromWishlist(wishlistId: number, userId: number): Promise<void>;
```

Reply:
해당 스타일로 수정해두겠습니다!